bitnami/redis-cluster/7.4/debian-12/prebuildfs/opt/bitnami/scripts/liblog.sh

```diff
@@ -39,7 +39,16 @@ stderr_print() {
 #   None
 #########################
 log() {
-    stderr_print "${CYAN}${MODULE:-} ${MAGENTA}$(date "+%T.%2N ")${RESET}${*}"
+    local prefix=""
+    local suffix=""
+    local color_check="${BITNAMI_COLOR:-true}"
+    if [[ "$color_check" =~ ^(yes|true)$ ]]; then
+        prefix="${CYAN}${MODULE:-} ${MAGENTA}$(date "+%T.%2N ")${RESET}"
+        suffix="${RESET}"
+    else
+        prefix="${MODULE:-} $(date "+%T.%2N ")"
+    fi
+    stderr_print "${prefix}${*}${suffix}"
 }
```

Comment:
I think a couple of minor changes in this method would make it clear, what do you think?

* Rename `color_check` to `color_bool`.
* Update `if` logic to match the one in line 29 of this file due to the reasons in line 25.